packages/Parallel/Application/ParallelFileProcessor.php

```diff
@@ -62,6 +62,9 @@ public function process(
         InputInterface $input
     ): array {
         $jobs = array_reverse($schedule->getJobs());
+        $jobIdentifiers = [];
+        $jobProcesses = [];
```

Comment:
In [last force-push diff](https://github.com/rectorphp/rector-src/compare/fc36aa17e6908fa0662547b7275c279f7539cab3..de3ae27708cb6169b6c252cb57c2fe87b27aedcd) I had to introduce this because I found an issue when a process timeout, `$this->processPool->quitAll();` is called, and it breaks when having attached processes that are not started yet.
So I can't use the process pool to store the job processes